esx_service/vsan_policy.py

```diff
@@ -87,6 +89,7 @@ def delete(name):
     try:
         os.remove(policy_path(name))
     except:
+        logging.exception("Failed to remove %s policy file", name)
```

Comment:
Yes, thought about it and decided to leave it there just in case for other OSErrors.